lib/js/components/DatePickers/DatePicker/DatePicker.vue

```diff
@@ -0,0 +1,278 @@
+<template>
+	<div
+		class="ds-datePicker"
+		:class="{ '-ds-box': triggerType === DATE_PICKER_TRIGGER_TYPES.BOX }"
+	>
+		<template v-if="triggerType === DATE_PICKER_TRIGGER_TYPES.TILE">
+			<span
+				class="ds-datePicker__label"
+				:class="{
+					'-ds-isUppercase': isLabelUppercase,
+					'-ds-disabled': state === DATE_PICKER_STATES.DISABLED,
+				}"
+				>{{ label }}</span
+			>
+			<ds-tile
+				:text="text"
+				:interactive="isInteractive"
+				:color="color"
+				:state="state"
+				:icon-right="icon"
+				:is-icon-right-hidden-on-mobile="isIconHiddenOnMobile"
+				:eyebrow-text="eyebrowText"
+				@click="toggle"
+			/>
+		</template>
+		<date-picker-box
+			v-else
+			:is-interactive="isInteractive"
+			:placeholder="placeholder"
+			:date="date"
+			:icon="icon"
+			:is-icon-hidden-on-mobile="isIconHiddenOnMobile"
+			:state="state"
+			:color="color"
+			:eyebrow-text="eyebrowText"
+			:is-open="isOpen"
+			@click="toggle"
+		/>
+
+		<span class="ds-datePicker__errorMessage"> {{ errorMessage }} </span>
+		<input ref="flatpickrInput" class="ds-datePicker__hiddenInput" />
+	</div>
+</template>
+
+<style lang="scss">
+@import '../../../../styles/settings/typography/tokens';
+@import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/spacings';
+@import 'flatpickr/dist/flatpickr';
+
+.ds-datePicker {
+	$self: &;
+
+	display: flex;
+	flex-direction: column;
+	row-gap: $space-4xs;
+
+	&.-ds-box {
+		display: inline-flex;
+	}
+
+	&__hiddenInput {
+		display: none;
+	}
+
+	&__label {
+		@include label-l-default-bold;
+
+		color: $color-neutral-text-strong;
+		padding-bottom: $space-4xs;
+
+		&.-ds-isUppercase {
+			@include label-m-extensive-bold-uppercase;
```

Comment:
is it intentional that the regular label has a different font than the uppercase?

Reply:
yes https://www.figma.com/design/izQdYyiBR1GQgFkaOIfIJI/LMS---DS-Components?node-id=8505-17297&m=dev